2024-03-26 12:40:09,283 - distributed.nanny - INFO -         Start Nanny at: 'tcp://172.16.3.184:33167'
2024-03-26 12:40:09,873 - distributed.diskutils - INFO - Found stale lock file and directory '/tmp/dask-worker-space/worker-n1gr4k06', purging
2024-03-26 12:40:11,009 - distributed.worker - INFO -       Start worker at:   tcp://172.16.3.184:35332
2024-03-26 12:40:11,010 - distributed.worker - INFO -          Listening to:   tcp://172.16.3.184:35332
2024-03-26 12:40:11,010 - distributed.worker - INFO -           Worker name:             SLURMCluster-9
2024-03-26 12:40:11,010 - distributed.worker - INFO -          dashboard at:         172.16.3.184:44123
2024-03-26 12:40:11,010 - distributed.worker - INFO - Waiting to connect to:    tcp://172.16.3.92:46856
2024-03-26 12:40:11,010 - distributed.worker - INFO - -------------------------------------------------
2024-03-26 12:40:11,010 - distributed.worker - INFO -               Threads:                         36
2024-03-26 12:40:11,010 - distributed.worker - INFO -                Memory:                 178.81 GiB
2024-03-26 12:40:11,010 - distributed.worker - INFO -       Local Directory: /tmp/dask-worker-space/worker-qjhvi1np
2024-03-26 12:40:11,010 - distributed.worker - INFO - -------------------------------------------------
2024-03-26 12:40:11,024 - distributed.worker - INFO -         Registered to:    tcp://172.16.3.92:46856
2024-03-26 12:40:11,024 - distributed.worker - INFO - -------------------------------------------------
2024-03-26 12:40:11,025 - distributed.core - INFO - Starting established connection to tcp://172.16.3.92:46856
2024-03-26 12:58:36,016 - distributed.core - INFO - Removing comms to tcp://172.16.3.194:35468
2024-03-26 13:17:05,714 - distributed.core - INFO - Removing comms to tcp://172.16.3.208:42872
2024-03-26 13:17:05,720 - distributed.core - INFO - Removing comms to tcp://172.16.3.123:35109
2024-03-26 13:17:05,720 - distributed.core - INFO - Removing comms to tcp://172.16.3.210:40902
2024-03-26 13:17:05,731 - distributed.core - INFO - Removing comms to tcp://172.16.3.207:43021
2024-03-26 13:17:05,731 - distributed.core - INFO - Removing comms to tcp://172.16.3.211:38939
2024-03-26 13:17:05,735 - distributed.core - INFO - Removing comms to tcp://172.16.3.209:41973
2024-03-26 13:17:05,743 - distributed.core - INFO - Removing comms to tcp://172.16.3.213:44279
2024-03-26 13:17:05,744 - distributed.core - INFO - Removing comms to tcp://172.16.3.212:35658
2024-03-26 13:31:05,687 - distributed.core - INFO - Removing comms to tcp://172.16.3.109:46525
2024-03-26 14:09:35,545 - distributed.core - INFO - Removing comms to tcp://172.16.3.195:44558
2024-03-26 14:30:35,071 - distributed.worker - ERROR - Failed to communicate with scheduler during heartbeat.
Traceback (most recent call last):
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/tornado/iostream.py", line 861, in _read_to_buffer
    bytes_read = self.read_from_fd(buf)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/tornado/iostream.py", line 1116, in read_from_fd
    return self.socket.recv_into(buf, len(buf))
ConnectionResetError: [Errno 104] Connection reset by peer

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/worker.py", line 1237, in heartbeat
    response = await retry_operation(
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/utils_comm.py", line 434, in retry_operation
    return await retry(
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/utils_comm.py", line 413, in retry
    return await coro()
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/core.py", line 1269, in send_recv_from_rpc
    return await send_recv(comm=comm, op=key, **kwargs)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/core.py", line 1028, in send_recv
    response = await comm.read(deserializers=deserializers)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/comm/tcp.py", line 241, in read
    convert_stream_closed_error(self, e)
  File "/vortexfs1/home/anthony.meza/mambaforge/envs/atm_rivers/lib/python3.10/site-packages/distributed/comm/tcp.py", line 142, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc.__class__.__name__}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.heartbeat_worker local=tcp://172.16.3.184:37805 remote=tcp://172.16.3.92:46856>: ConnectionResetError: [Errno 104] Connection reset by peer
2024-03-26 14:30:35,076 - distributed.core - INFO - Connection to tcp://172.16.3.92:46856 has been closed.
2024-03-26 14:30:35,076 - distributed.worker - INFO - Stopping worker at tcp://172.16.3.184:35332. Reason: worker-handle-scheduler-connection-broken
2024-03-26 14:30:35,078 - distributed.nanny - INFO - Closing Nanny gracefully at 'tcp://172.16.3.184:33167'. Reason: worker-handle-scheduler-connection-broken
2024-03-26 14:30:35,080 - distributed.nanny - INFO - Worker closed
2024-03-26 14:30:37,260 - distributed.nanny - INFO - Closing Nanny at 'tcp://172.16.3.184:33167'. Reason: nanny-close-gracefully
2024-03-26 14:30:37,261 - distributed.dask_worker - INFO - End worker
